eng/build.sh

```diff
@@ -286,6 +286,14 @@ while [[ $# > 0 ]]; do
           os="illumos" ;;
         solaris)
           os="solaris" ;;
+        linux-bionic)
+          os="linux"
```

Comment:
should this set `linux-bionic` like musl does?

Reply:
The `linux-musl` one is wrong. For `linux-musl` and `linux-bionic` `os` (which is `TargetOS` in the props file) is `linux`.